src/main/java/org/example/HomeworkDao.java

```
@@ -0,0 +1,25 @@
+package org.example;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+public class HomeworkDao {
+    public void addHomework(Homework homework) {
+        String query = "INSERT INTO Homework (name, description) VALUES (?, ?)";
+        int nameIndex = 1;
+        int descriptionIndex = 2;
+
+        try (Connection connection = DataBaseConnection.getConnection();
+             PreparedStatement statement = connection.prepareStatement(query)) {
+
+            statement.setString(nameIndex, homework.getName());
+            statement.setString(descriptionIndex, homework.getDescription());
+
+            statement.executeUpdate();
+        } catch (SQLException e) {
+            System.err.println("Error adding homework: " + e.getMessage());
```

What is that? Haven't I told you that you would better throw an exception instead of using a print statement?